style.css

```diff
@@ -74,3 +74,371 @@ li {
 .icon {
   margin: 7px;
 }
+
+.IndicatorIndi {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 15%;
+  margin-top: 5%;
+  width: 15%;
+}
+
+/* GRID =========================== */
+.grid-container {
+  max-width: 327px;
+  height: 474px;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 5fr 2fr 1fr 2fr;
+  background-color: #ebf0ee;
+  border-color: #ebf0ee;
+  border-width: 50px;
+  border-radius: 8px;
+  border: 1px solid #c1c7d0;
+  margin-top: 10%;
+}
+
+.grid-item {
+  border-radius: 8px;
+}
+
+.main {
+  background-color: #fff;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+h3 {
+  font-family: Inter, sans-serif;
+  text-align: center;
+  font-size: 24px;
+}
+
+/* FLEXBOX ====================== */
+.flex-container {
+  display: flex;
+  gap: 10px;
+  background-color: #fff;
+  border-bottom-left-radius: 0%;
+  border-bottom-right-radius: 0%;
+}
+
+.flex-item {
+  flex: 1;
+  border-radius: 8px;
+  color: #3a4a42;
+}
+
+.flex-item1 {
+  background-color: #ebf0ee;
+  flex: 2;
+  width: 90px;
+  height: 32px;
+  border-radius: 8px;
+  font-family: Inter, sans-serif;
+  font-size: 10px;
+  text-align: center;
+  color: #3a4a42;
+  line-height: 16px;
+  border-bottom-left-radius: 4%;
+}
+
+.flex-item2 {
+  background-color: #ebf0ee;
+  height: 32px;
+  font-family: Inter, sans-serif;
+  font-size: 10px;
+  text-align: center;
+  color: #3a4a42;
+  line-height: 16px;
+}
+
+.flex-item3 {
+  background-color: #ebf0ee;
+  height: 32px;
+  font-family: Inter, sans-serif;
+  font-size: 10px;
+  text-align: center;
+  color: #3a4a42;
+  line-height: 16px;
+}
+
+.flex-item4 {
+  background-color: #ebf0ee;
+  height: 32px;
+  font-family: Inter, sans-serif;
+  font-size: 10px;
+  text-align: center;
+  color: #3a4a42;
+  line-height: 16px;
+  border-bottom-right-radius: 4%;
+}
+
+h4 {
+  margin-top: 8px;
+}
+
+.button1 {
+  color: black;
+  border: 2px solid #4caf50;
+  background-color: #36b37f;
+  border-radius: 2px;
+  width: 122px;
+  height: 48px;
+  font-size: 17px;
+}
+
+.last {
+  background-color: #fff;
+  display: grid;
+  border-top-left-radius: 0%;
+  border-top-right-radius: 0%;
+}
+
+button {
+  background-color: lightblue;
+  margin: auto;
+  width: 122px;
+  height: 48px;
+}
+
+/* About Section */
+.First-image img {
+  float: right;
+}
+
+.grid-image {
+  max-width: 327px;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr;
+  border-width: 50px;
+  border-radius: 8px;
+}
+
+.button-resume {
+  color: black;
+  border: 2px solid #4caf50;
+  background-color: #36b37f;
+  border-radius: 2px;
+  width: 150px;
+  height: 48px;
+  font-size: 17px;
+  margin-top: 7%;
+}
```

Comment:
- [ ] Kindly implement the button correctly to match the given design. The `background color` of the button and `text color` is not matching with the given design.
- [ ] Also implement the button interactions such as (`hover`, `click`, `disable`)
- Here is the button interactions instruction:
- ![image](https://github.com/ishimwepati/Portfolio/assets/60083437/aa082b50-360f-44e4-805d-85e6f3396484)

- ***Your Design***
- ![image](https://github.com/ishimwepati/Portfolio/assets/60083437/81a1f964-979a-4032-b6c9-3feae15f873d)
- ***Given Design***
- ![image](https://github.com/ishimwepati/Portfolio/assets/60083437/0b540a11-9f42-4dfc-8265-c81519d73428)